### PR TITLE
fix: code style of source files

### PR DESCRIPTION
**Pull request by bednar**
## Proposed Changes

This PR enable CI check for source file code style checking.

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] Tests pass
- [x] Commit messages are [conventional](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

**Comment by bednar** [comment]
@karel-rehor, the Codecov report isn't applicable in this case due to the refactoring of the file styles.

File: Examples/batching_example.py
Comment (karel-rehor) [inline Examples/batching_example.py:1]: Ran this example against an initial user account - it eventually resulted in the following error

```
The retriable error occurred during request. Reason: 'org fbb67cd4c8905be6 has exceeded limited_write plan limit'. Retry in 206s.
```

File: Examples/cloud_dedicated_query.py
Comment (karel-rehor) [inline Examples/cloud_dedicated_query.py:1]: Tried this example as well against a new user account.  Apparently the "flight2" bucket needs to be created in another example.  

Failed with error
```
pyarrow.lib.ArrowInvalid: Flight returned invalid argument error, with message: bucket "flight2" not found. gRPC client debug context: UNKNOWN:Error received from peer ipv4:34.196.233.7:443 {created_time:"2024-04-05T10:59:19.006603056+02:00", grpc_status:3, grpc_message:"bucket \"flight2\" not found"}. Client context: IOError: Server never sent a data message. Detail: Internal
```

File: Examples/batching_example.py
Comment (bednar) [reply Examples/batching_example.py:1]: This issue should be addressed by https://github.com/InfluxCommunity/influxdb3-python/issues/82

File: Examples/cloud_dedicated_query.py
Comment (bednar) [reply Examples/cloud_dedicated_query.py:1]: This issue should be addressed by https://github.com/InfluxCommunity/influxdb3-python/issues/82

**Review by karel-rehor — APPROVED** [review]
* Tests pass.  Could be more of them.  
* Linter (flake8) passes. 
* Tried running a few examples but they have issues.
   * Not self contained - some expect a prepared database state
   * One exceeded limits for introductory account
   * Some use outdated API, which has been refactored
   * Still with a few changes they can be executed
   * For these issues Enhancement Request #82 has been opened
 
Since this PR is basically about standardizing code style and removing detritus like unused import statements, it should not be necessary to address issues with examples here.

Good to go.

**Pull request merged**